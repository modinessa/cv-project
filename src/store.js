import { configureStore } from "@reduxjs/toolkit";
import { navigationPanelReducer } from "./components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationPanelReducer,
	}
});
