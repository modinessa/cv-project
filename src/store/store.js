import { configureStore } from "@reduxjs/toolkit";
import navigationReducer from "../components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationReducer,
	}
});