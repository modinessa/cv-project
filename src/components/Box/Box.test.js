import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import { store } from '../../store/store';
import { Box } from './index';

const TEST_TITLE = 'Title';
const TEST_CONTENT = 'Content bla-bla';

describe('Box', () => {
	it('renders component', () => {
		expect(renderer.create(
			<Provider store={store}>
				<Box
					title={TEST_TITLE}
					content={TEST_CONTENT}
				/>
			</Provider>
		)).toMatchSnapshot();
	})
})