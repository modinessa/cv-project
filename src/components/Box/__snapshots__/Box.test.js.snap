// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box renders component 1`] = `
<section
  className="app-section"
>
  <h2
    className="app-title"
  >
    Title
  </h2>
  <p>
    Content bla-bla
  </p>
</section>
`;
